packages/components/organisms/f-form/package.json

```diff
@@ -43,10 +43,14 @@
     "@justeat/f-services": "1.7.0"
   },
   "peerDependencies": {
-    "@justeat/browserslist-config-fozzie": ">=1.2.0"
+    "@justeat/browserslist-config-fozzie": ">=1.2.0",
+    "@justeat/f-button": "3.x.x",
```

Comment:
What's the need to have button and form-field as both dev and peer dependencies?

Reply:
This is something Ash recommended here https://github.com/justeat/fozzie-components/pull/1554#discussion_r779624384.
It's in peer so they don't get built as part of the f-form component
Then it's in dev so that the tests can still run with no problems